test/unit/configuration_test.rb

```diff
@@ -71,6 +74,26 @@ def test_empty_config_file_doesnt_break_log_level
         assert_equal :info, config.log_level, 'should be default'
       end
 
+      def test_empty_config_file_doesnt_break_suites
+        config = fixture_config('empty')
+        assert_equal '', config.suites, 'should be default'
+      end
+
+      def test_default_suites
+        c = Configuration.new
+        assert_includes c.send(:default_suites), :rack
```

Comment:
you might retrieve the default_suites once and then assert on the result to DRY this up?